e2e/app/page/workspace-base.ts

```diff
@@ -222,20 +222,22 @@ export default abstract class WorkspaceBase extends AuthenticatedPage {
   }
 
   /**
-   * Select Workspace action dropdown menu option.
-   * @param {EllipsisMenuAction} menuOption
+   * Select Workspace action snowman menu option.
+   * @param {Option} option
    * @param opts
    */
-  async selectWorkspaceAction(menuOption: EllipsisMenuAction, opts?: { waitForNav: false }): Promise<void> {
-    const ellipsisMenu = new EllipsisMenu(this.page, './/*[@data-test-id="workspace-menu-button"]');
-    return ellipsisMenu.clickAction(menuOption, opts);
+  async selectWorkspaceAction(option: Option, opts?: { waitForNav: false }): Promise<void> {
+    const iconXpath = './/*[@data-test-id="workspace-menu-button"]';
```

Comment:
how come you're accessing this this way instead of the usual way of looking for the ellipsis icon?

Reply:
Icon is not standard ellipsis icon. xpath selector is different from standard three-dots.
![Screen Shot 2020-09-30 at 12 15 00 PM](https://user-images.githubusercontent.com/35533885/94711877-b5408380-0316-11eb-9456-3e1179943c54.png)

Reply:
Argh, of course it's a fontawesome icon here and a clarity icon everywhere else

Reply:
An RW ticket should be created to change this icon into a clarity icon (or all Snowman icons into fontawesome icons); would you like to do that or would you like me to do that?

Reply:
Filed new ticket [RW-5682](https://precisionmedicineinitiative.atlassian.net/browse/RW-5682)

Reply:
thank you!